libs/zephyr-agent/README.md

```diff
@@ -95,6 +95,98 @@ interface ZephyrAgentConfig {
 }
 ```
 
+### Git Repository Requirements
+
+**IMPORTANT**: Zephyr requires a properly initialized Git repository with a remote origin for production deployments.
+
+#### Git Information Handling
+
+When Zephyr cannot find a Git repository with remote origin, it will:
+
+1. **Automatic Package.json-Based Naming**:
+
+   - Extract organization, project, and app names from your `package.json`
+   - Use authenticated user's username as the organization for personal Zephyr org
+   - Follow intelligent naming conventions based on package structure
+   - No user prompts or environment variables required
+
+2. **Naming Logic**:
+   - **Scoped packages** (`@scope/name`): project = scope, app = name
+   - **With root package.json**: project = root package name, app = current package name
```

Comment:
I have forgot one use case, if root package is scoped, root package has scope and app doesn't - use scope from root

Reply:
Sure, added